src/pages/articles/bootstrap/bootstrap-get-started/index.md

```diff
@@ -2,14 +2,28 @@
 title: Bootstrap Get Started
 ---
 ## Bootstrap Get Started
+When using Bootstrap for the first time it can appear to be a confusing process, but using Bootstrap is actually quite simple. Using Bootstrap makes it easy to design a fully responsive website and is a framework worth learning.
```

Comment:
Try not to use simple, it can have a negative effect on a reader that is struggling